### PR TITLE
Diego/ora 1481 stress tests add epochlength as option

### PR DESCRIPTION
**Pull request by xmariachi**
Add EpochLength as env var

### CODE REVIEW

**Comment by br4e** [comment]
Please merge into major-upgrade, NOT main

**Review by br4e — APPROVED** [review]

File: test/stress/n_workers_reputers__stress_test.go [hunk 1]
```diff
@@ -375,9 +374,8 @@ func WorkerReputerCoordinationLoop(m TestMetadata, reputersPerEpoch, reputersMax
 	var wg sync.WaitGroup
 	if topicsPerEpoch == 0 {
 		topicFunderAddress, topicFunderAccount := getTopicFunder()
-		wg.Add(1)
```
[the remark this answers is not in the document]
Comment (relyt29) [reply test/stress/n_workers_reputers__stress_test.go:378]: Don't you need a wait group for every separate thread / channel you launch

[the remark this answers is not in the document]
Comment (br4e) [reply test/stress/n_workers_reputers__stress_test.go:378]: yes but not if you are only launching one topic, you can just use the main thread

[the remark this answers is not in the document]
Comment (xmariachi) [reply test/stress/n_workers_reputers__stress_test.go:378]: That code was nonconsistent as it was: if it's only that thread, the wg could be moved to inside the `else`.
But just to make it uniform, I've made it a subroutine too.

**Review by relyt29 — APPROVED** [review]
i'm good with it, even though it's a bit silly to spin off a thread when you don't need one

**Pull request merged**